models/autotetraploid.slim

```diff
@@ -0,0 +1,134 @@
+// Peter Ralph, 10 September 2019: autotetraploidy
+
+/*
+	Tetraploidy:
+	
+	This is a simple model of an autotetraploid.  To do this, we have to 
+	(a) give each individual *four* genomes
+	(b) let crossing-over happen between any of the possible pairings
+	(c) compute fitness with all four genomes
+	
+	(a) Four genomes: the "real" individuals are in p1. Each individual has their other two genomes stashed
+	away in an individual in p2; the individuals in p2 don't do anything on their own. To identify them,
+	each individual in p1 has a "tag" that records the index of their pair in p2, so that
+	  p2.individuals[ind.tag].genomes
+	gets you the other two genomes for individual "ind" in p1. (To create these associations, we give each
+	individual a unique label, stored in their tagF property.) After an individual dies, we clean up their
+	paired individual by setting their fitness to 0.0 in the following generation. So, there's always more
+	individuals in p2 than in p1; the difference is individuals in p2 whose pair in p1 has just died.
+	
+	(b) For crossing-over, we pick an ordering of the mates' four genomes, then do crossing-over by picking
+	a Poisson number of uniform recombination points on each of the four pairs, with an extra crossing-over
+	at 0.0 to randomize which genome we start with. If we want non-uniform recombination, we'd have to get
+	these recombinations some other way.
```

Comment:
This (b) note needs to be revised now that you're using drawBreakpoints().  Apart from that, the PR looks ready to go, thanks!